src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -195,10 +195,15 @@ export class FacetValueSuggestions extends Component {
     const wordsToQuery = this.options.useValueFromSearchbox ? [text] : [];
 
     const suggestionsKeywords: string[] = await this.getQuerySuggestionsKeywords(omnibox);
-    const allWordsToQuery = _.unique(wordsToQuery.concat(suggestionsKeywords));
+    const allWordsToQuery = _.unique(wordsToQuery.concat(suggestionsKeywords).filter(value => value != ''));
```

Comment:
Could we simply change the "getSuggestion" method so that it becomes : 

if text == "" return [] ?

Seems a bit more straightforward to read and generally simpler. Plus, does not have to wait to call to the "getQuerySuggestionsKeywords" method.

Reply:
I'd rather put `if (allWordsToQuery.length == 0) return [];` in this method.

Query box can be empty, but still provide ML query suggestions, which can be used by this component.